### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by xpepermint**
[] Fix tests

Refs:
- https://github.com/nodejs/node-gyp/issues/1862
- https://github.com/nodejs/node-gyp/issues/1867

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,19 +8,19 @@ We use [RushJS](https://rushjs.io) to manage this repository. Some quick notes o
 
 **Install dependencies** -- You only need to run this once.
 
-```sh
+```
```
Comment (fulldecent) [inline CONTRIBUTING.md]: 😠

Comment (fulldecent) [reply CONTRIBUTING.md]: Lost the semantics here. This is not just _any_ code, it's Borne Shell code!

Comment (xpepermint) [reply CONTRIBUTING.md]: True but `sh` does not work in any editor. That's why I removed it ;).

Comment (fulldecent) [reply CONTRIBUTING.md]: Here is the editor you want to be using for Markdown!

https://typora.io

**Pull request merged**